orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -895,7 +898,8 @@ Status TrainingRunner::TrainingLoop(IDataLoader& training_data_loader, IDataLoad
             << "Total Running Time: " << total_time << " Seconds \n"
             << "Average Running Time Per Batch: " << avg_time_per_batch << " ms\n"
             << "Throughput: " << throughput << " Examples / Second\n"
```

Comment:
Now we have 3 throughput - should we put comments on top what's the different among them?
Also, I am populating 2 throughputs to MySQL DB->PowerBI report (below), should this new entry (e2e) be added as well?

Reply:
use the EndToEnd Throughput.

Reply:
so discard the other 2 TP (for perf monitor), just use e2e TP then (after this is merged)? 

---
In reply to: [431491842](https://github.com/microsoft/onnxruntime/pull/4049#discussion_r431491842) [](ancestors = 431491842)

Reply:
BTY - the bert perf test only run 100 steps - looks like we need to extend it to 200 so it's > 128 

---
In reply to: [431522622](https://github.com/microsoft/onnxruntime/pull/4049#discussion_r431522622) [](ancestors = 431522622,431491842)

Reply:
right... we can have it run longer, say 512+128 steps

Reply:
sure, will do the same for gpt-2 then